live_data/websocket_subscriber/__init__.py

```diff
@@ -0,0 +1,6 @@
+from live_data.websocket_subscriber.live_data_consumer import LiveDataConsumer
+
+
+def create_live_data_consumer(device_api_client_mapping):
+    print(f"create_live_data_consumer")
+    live_data = LiveDataConsumer(device_api_client_mapping)
```

Comment:
This is also not needed, a `lambda: LiveDataConsumer(device_api_client_mapping)` inside the main.py should suffice if you want to defer execution of this method because it is blocking.